### PR TITLE
Add essential SEO metadata to each file

### PR DESCRIPTION
**Pull request by MihaiB-dev**
Verify if the title, description and keywords are good for each file

### CODE REVIEW

**Comment by MihaiB-dev** [comment]
plis plis plis

File: docs/intro.md
Comment (sirbuig) [inline docs/intro.md:1]: We have to change lines 8-15 content as well

File: docs/week-3/problems/ex1.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
 sidebar_position: 1
+title: Exercise 1 Size of Files
```
Comment (sirbuig) [inline docs/week-3/problems/ex1.md:3]:
```suggestion
title: Exercise 1 - Size of Files
```

File: docs/week-3/problems/ex2.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ---
 sidebar_position: 2
+title: Exercise 2 - Search word
```
Comment (sirbuig) [inline docs/week-3/problems/ex2.md:3]:
```suggestion
title: Exercise 2 - Search the word

```

File: docs/week-3/problems/ex2.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 ---
 sidebar_position: 2
+title: Exercise 2 - Search word
+description: Search for a specific string in multiple text files
+keywords: [Buffer, String Search, File Handling, C Programming, Text Processing, Directory Traversal, Edge Cases, File I/O, fopen, opendir, readdir]
 ---
 
 # Exercise 2 (hard) - Search word
```
Comment (sirbuig) [inline docs/week-3/problems/ex2.md:8]:
```suggestion
# Exercise 2 (hard) - Search the word

```

File: docs/weeks-1-2/linux-basics/tutorial.mdx [hunk 4]
```diff
@@ -1,5 +1,8 @@
 ---
 sidebar_position: 2
+title: Linux Basics Tutorial
```
Comment (sirbuig) [inline docs/weeks-1-2/linux-basics/tutorial.mdx:3]:
```suggestion
title: Get familiar with the Linux CLI
```

**Review by sirbuig — CHANGES_REQUESTED** [review]
:man_technologist:

File: docs/intro.md
Comment (MihaiB-dev) [reply docs/intro.md:1]: What do you mean?

**Review by MihaiB-dev — COMMENTED** [review]
good

**Review by sirbuig — APPROVED** [review]

**Pull request merged**